gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

Comment:
Could you run `./gradlew wrapper`, such that `gradlew` and `gradlew.bat` etc. are updated correctly?

Reply:
Shell scripts weren't updated, but I guess i can update the wrapper jar prematurely